### PR TITLE
able to delete tournaments and brackets

### PR DESCRIPTION
**Pull request by joram**

### CODE REVIEW

File: play/apps/tournament/templates/tournament_bracket/show.html [hunk 1]
```diff
@@ -52,5 +52,20 @@
...
 </div>
+
+
+<div class="card">
+  <div class="card-header">Snakes in Bracket</div>
+  <div class="card-body container">
+    <ul class="snakes">
+    {% for snake in tournament_bracket.snakes %}
+      <li class="snakes">
+        {{ snake.name }} ({{ snake.team.name }})
```
Comment (joram) [inline play/apps/tournament/templates/tournament_bracket/show.html]: this won't show the team until we merge the refactor branch

File: play/apps/tournament/views/tournament.py [hunk 2]
```diff
@@ -39,16 +39,24 @@ def new(request):
...
-        form = TournamentForm(instance=tournament_group)
+        form = TournamentForm(instance=tournament)
 
     return render(request, 'tournament/edit.html', { 'form': form })
+
+
+@admin_required
+@login_required
+def delete(request, id):
+    tournament = Tournament.objects.get(id=id)
```
Comment (brandonb927) [inline play/apps/tournament/views/tournament.py]: You could just call `.delete` here. Also, would be nice to send a success `message` that gets displayed in the UI after deleting the tournament

File: play/apps/tournament/views/tournament_bracket.py [hunk 3]
```diff
@@ -41,6 +41,15 @@ def edit(request, id):
     return render(request, 'tournament_bracket/edit.html', {'form': form})
 
 
+@admin_required
+@login_required
+def delete(request, id):
+    tournament_bracket = TournamentBracket.objects.get(id=id)
+    tournament_id = tournament_bracket.tournament.id
+    tournament_bracket.delete()
+    return redirect(f'/tournament/{tournament_id}')
```
Comment (brandonb927) [inline play/apps/tournament/views/tournament_bracket.py]: would be nice to see the success `message` here too

**Comment by brandonb927** [comment]
protip: destructive actions like deleting resources should have confirmations, especially Tournaments :)

**Comment by joram** [comment]
this one is very otu of date.

**Pull request closed without merging**